#1 [internal] load local bake definitions
#1 reading from stdin 1.36kB done
#1 DONE 0.0s

#2 [nginx internal] load build definition from Dockerfile.nginx
#2 transferring dockerfile: 89B done
#2 DONE 0.0s

#3 [coldfusion internal] load build definition from Dockerfile.coldfusion
#3 transferring dockerfile: 135B done
#3 DONE 0.0s

#4 [coldfusion internal] load metadata for docker.io/adobe/coldfusion:2018-update16
#4 ...

#5 [nginx internal] load metadata for docker.io/library/nginx:1.27.0
#5 DONE 0.2s

#6 [nginx internal] load .dockerignore
#6 transferring context: 2B done
#6 DONE 0.0s

#7 [nginx 1/2] FROM docker.io/library/nginx:1.27.0@sha256:98f8ec75657d21b924fe4f69b6b9bff2f6550ea48838af479d8894a852000e40
#7 resolve docker.io/library/nginx:1.27.0@sha256:98f8ec75657d21b924fe4f69b6b9bff2f6550ea48838af479d8894a852000e40 0.0s done
#7 DONE 0.1s

#4 [coldfusion internal] load metadata for docker.io/adobe/coldfusion:2018-update16
#4 ERROR: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed

#7 [nginx 1/2] FROM docker.io/library/nginx:1.27.0@sha256:98f8ec75657d21b924fe4f69b6b9bff2f6550ea48838af479d8894a852000e40
#7 sha256:ed60582f2661431435c1a1c94e7573d69357d633961280cebaf6aac9988afa5e 0B / 1.40kB 0.3s
#7 sha256:472851929ad734a0bbf2e262059d7d1521d7c1904af3b10e520e18b372e3455c 0B / 1.21kB 0.3s
#7 sha256:ca3474dd2ca6ca60337319448fe246d55f7ed8f3078b31132924b325f3859f8d 0B / 394B 0.3s
#7 sha256:e9d00ba15cc0967faaecc85ea2acdc033a103a018a26b09e54479d344ad06674 0B / 955B 0.2s
#7 DONE 0.4s

#8 [nginx 2/2] WORKDIR /etc/nginx
#8 CANCELED
------
 > [coldfusion internal] load metadata for docker.io/adobe/coldfusion:2018-update16:
------
Dockerfile.coldfusion:1

--------------------

   1 | >>> FROM adobe/coldfusion:2018-update16

   2 |     ENV CFADMIN_PASSWORD=admin

   3 |     EXPOSE 8500

--------------------

target coldfusion: failed to solve: adobe/coldfusion:2018-update16: failed to resolve source metadata for docker.io/adobe/coldfusion:2018-update16: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed

